### DISCUSSION TITLE
[Bug]: Only Medium Layout works

### DISCUSSION THREAD

**Discussion by buble** [mikekazakov/nimble-commander#334] [Q&A] [answered]
### Confirmation

- [X] I have read the contributing guidelines

### Version of Nimble Commander

v 1.6.0 macOS 14.5

### Distribution Type

Downloaded directly from the website (including Brew)

### Expected Result

All Layout should work

### Actual Result

Can't switch to other layouts
Short, Full and Wide layouts seems not working at all

### Steps to reproduce

Go to Settings->Panels->Layout
Try to change Layout

### Additional Information

![layout_bug](https://github.com/user-attachments/assets/dee4a08d-f0fc-4cc1-8737-2d7f595367a8)

**Comment by mikekazakov** [comment]
Moving to Discussions.

**Comment by mikekazakov** [comment, marked as answer]
Switching between different layouts in the Preferences/Settings dialog doesn't change the actual layout that a file panel uses.
Instead it chooses what layout should be edited in that dialog.
To change a view mode of a file panel use Menu -> View -> View Mode -> ... or hotkeys: Ctrl+1, Ctrl+2, Ctrl+3, etc.
